### ISSUE TITLE
'More' buttons are not descriptive

### ISSUE THREAD

**Issue by jon-kirwan** [alphagov/govuk_publishing_components#2990]
## What:

The "**+ 3 more**" button from the [related navigation component](https://components.publishing.service.gov.uk/component-guide/related_navigation) is not descriptive enough to identify its function or purpose when navigating out of context.

**Example URLs:**

* https://www.gov.uk/check-uk-visa
* https://www.gov.uk/cymraeg

A [previous similar issue was solved](https://github.com/alphagov/govuk_publishing_components/pull/2866) by removing the functionality altogether and always displaying all items.

## Why:

Confusing or missing descriptions don't provide enough context for all users, including screen reader users.

## Prioritisation score: 4

Reach: `2`, Severity: `2`, Effort: `1`
